src/puppet-mock.ts

```diff
@@ -294,11 +313,8 @@ class PuppetMock extends Puppet {
     return this.mocker.messagePayload(id)
   }
 
-  public async messageSendText (
-    conversationId: string,
-    text     : string,
-  ): Promise<void> {
-    log.verbose('PuppetMock', 'messageSend(%s, %s)', conversationId, text)
+  private async messageSend (conversationId: string, something: any): Promise<void> {
```

Comment:
1. Please keep the code format (each argument for one line) in your PR
1. Please do not use `any`: define specific typing for this `something` argument because it is necessary to make sure the code quality in the future.